src/picosvg/svg_meta.py

```diff
@@ -142,7 +142,7 @@ def parse_css_declarations(
     output: MutableMapping[str, Any],
     property_names: Optional[Container[str]] = None,
 ) -> str:
-    """Parse CSS declaration list into {property: value} dict.
+    """Parse CSS declaration list into {property: value} dict. Vendor styles (starting with a dash) are ignored even when provided in property_names as the XML etree module does not support attribute names starting with dash.
```

Comment:
the line is a bit too long. Keep the original initial sentence on its own line, the add the rest to the following lines (perhaps keep an empty line separating the first from the rest of the lines) and try to stay within 88 characters if you can